cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/UpdateController.java

```diff
@@ -139,7 +193,12 @@ public Integer call() throws Exception {
 
             DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
             ApplicationsOperations applicationsOperations = new ApplicationsOperations(cfOperations);
-            applicationBeans.keySet().forEach(applicationsOperations::removeApplication);
+
+            cfOperations.getOrganizationId().block();
+            Flux.fromIterable(applicationBeans.keySet())
+                    .flatMap(applicationsOperations::remove)
+                    .onErrorContinue(Log::warning)
+                    .blockLast();
 
             return 0;
```

Comment:
We should return 1 if there were any problems during removal

Reply:
I think this is also done in the BaseController. Not sure with the onErrorContinue method.